### DISCUSSION TITLE
Download failed, error code: null

### DISCUSSION THREAD

**Discussion by fmnxl** [zepp-health/zeppos-forum#264] [Q&A]
I'm trying to run the Hello World example on my watch (Bip 3 Pro). I've generated the QR code with `zeus preview`, which I then scan with the Zepp app inside the device's Developer mode, but I keep getting the "Download failed, error code: null" error.

I have tried adding `bip3` to the list of platforms in the app.json but nothing helped. I also cannot find the device ID on the list (https://docs.zepp.com/docs/v2/reference/related-resources/device-list/).

**Comment by silver-zepp** [comment]
Bip 3 doesn't have ZeppOS on board. You need at least Bip 5 from this family to be able to create your apps.
